src/components/chart/_chart.scss

```diff
@@ -88,3 +88,40 @@
 .highcharts-a11y-proxy-group-legend button {
     cursor: auto !important;
 }
+
+.ons-chart__range-annotation-label {
+    &--x {
+        padding: 0 20px;
```

Comment:
Shouldn't we be using rems?

Reply:
Doh. Thanks for spotting that. Now updated.